governance/program/tests/program_test/mod.rs

```diff
@@ -98,6 +102,8 @@ impl GovernanceProgramTest {
 
     #[allow(dead_code)]
     pub async fn start_with_voter_weight_addin() -> Self {
+        // We only ensure the add-in is built but it doesn't detect changes
+        // If the addin is changed it needs to be manually built
```

Comment:
If this is getting annoying, you want to use a `rerun-if-changed` directive from a build script: https://doc.rust-lang.org/cargo/reference/build-scripts.html

Reply:
We've tried this before but it didn't work because we have the chicken and egg problem here.
The `addin` depends on `spl-governance` and has to be built after it and not before.
We only need the `voter-weight-addin` as a programable mock program and it doesn't change much so I think it's acceptable for now.